pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -364,9 +367,13 @@ class ModelInterpreter implements Serializable {
                   Stage parentStage) {
         return {
             Utils.logToTaskListener(reason.message)
-            Utils.markStageWithTag(thisStage.name, StageStatus.TAG_NAME, reason.stageStatus, reason.isNotExecutedNode())
+            Utils.markStageWithTag(thisStage.name, StageStatus.TAG_NAME, reason.stageStatus)
```

Comment:
Any reason we couldn't do a bulk marking for both parallels and normal stages here, without checking for parallel content? Or maybe there's a subtlety I miss?

Reply:
I wanna make sure both the stage and the parallel branch it's in get marked correctly, 'cos I don't know for sure what `StatusAndTiming` will tell Blue Ocean otherwise. *shrug*